content/00.front-matter.md

```diff
@@ -57,9 +57,20 @@ on {{manubot.date}}.
 ## Contact information
 \*Correspondence: jaclyn.taroni@ccdatalab.org, rokita@chop.edu
 
+<!-- Cell Genomics author instructions: https://marlin-prod.literatumonline.com/pb-assets/journals/research/cell-genomics/cell-genomics_IfA.pdf -->
+
 ## In Brief
-<!-- to add -->
+<!-- aka the eTOC blurb. A short summary that describes the context and significance of the findings for the broader readership in 80 words or less. -->
+The OpenPBTA is a global, collaborative open-science initiative which brought together researchers and clinicians to genomically characterize over 1,000 pediatric brain tumors. Shapiro, et. al create over 40 open-source, scalable modules to perform cancer genomics analyses and provide a richly-annotated somatic dataset across 59 brain tumor histologies. The OpenPBTA framework can be used as a model for large-scale data integration to inform basic research, therapeutic target identification, and clinical translation. 
 
 ## Highlights
 <!-- Highlights are bullet points that convey the core findings of your paper. You may include up to four highlights. The length of each highlight cannot exceed 85 characters (including spaces).-->
+OpenPBTA collaborative analyses establish resource for 1,253 pediatric brain tumors
+
+NGS-based WHO-aligned integrated diagnoses generated for 1,260 of 2,840 biospecimens
+
+Classifiers identify medulloblastoma subgroups, TP53 status, and telomerase activity
```

Comment:
I just thought of this - telomerase activity isn't exactly a _classifier_ since its outcome is numeric, right?

Reply:
tp53 scores are also a numeric outcome, but i suppose that is trained to classify and uses a cutoff (0.5). but looking at the EXTEND paper a bit more, you are right, this is more of an algorithm applied using a gene expression signature, so I will update. thanks!